cakefuzzer/scanners/iteration_result.py

```diff
@@ -18,13 +20,17 @@ async def scan(
         registry: VulnerabilitiesAdd,
         result: IterationResult,
     ) -> None:
+        
+        outcome = find_html_location(result.output.output, self.phrase)
```

Comment:
I think it should be moved into `VulnerabilityBuilder` or `get_vulnerability_objects` method. 
We want to call this html finder with the same regex phrase that we use in the builder. We can talk about how to do that.